database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -12,12 +12,15 @@ class CreateUsersTable extends Migration {
 	 */
 	public function up()
 	{
-		Schema::create('users', function(Blueprint $table)
-		{
+		Schema::create('users', function (Blueprint $table) {
 			$table->increments('id');
+
+			$table->string('github_id')->unique();
 			$table->string('name');
-			$table->string('email')->unique();
-			$table->string('password', 60);
+			$table->string('email');
```

Comment:
I tried running this locally and it didn't bring back an email. Not sure why, but I think we need to make email and name nullable, basd on my experiences with giscus